10.md

```diff
@@ -48,6 +48,8 @@ Where:
 
 **The order of marked "e" tags is not relevant.**  Those marked with `"reply"` denote the `<reply-id>`.  Those marked with `"root"` denote the root id of the reply thread.
 
+A direct reply to the root of a thread should have a single marked "e" tag of type "reply".
+
```

Comment:
it's the opposite actually, a single reply should contain only an `e` marked root

Reply:
Updated PR.